libc-test/build.rs

```diff
@@ -324,6 +324,9 @@ fn test_apple(target: &str) {
             // macOs 12 minimum
             "backtrace_async" => true,
 
+            // deprecated api
+            "getcontext" => true,
```

Comment:
I'm not sure if adding a deprecated API is a good idea. If there are no good alternatives, users should report it upstream, adding it here just delays the problem.

Reply:
I hesitated to be honest.

Reply:
There isn't any good alternative afaik, and I'm not sure if Apple cares about low-level APIs enough to provide one

Reply:
My concern is that deprecated items are likely to be removed, i.e. they would introduce a breaking change. Such a code should belong to userland IMHO.

Reply:
@devnexen By the way, we don't have to ignore it on the test anymore, right?